tools/elastic/migration/ExampleHostedService.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace TNO.Elastic.ExampleHosted;
+
+/// <summary>
+/// ExampleHostedService class, provides an example hosted service that stops immediately.
+/// </summary>
+public class ExampleHostedService : IHostedService
```

Comment:
I found interesting information about hosted services.  After reading up on it I think I should eventually refactor the services I've built to become hosted services.